tensorflow/core/ops/math_ops.cc

```diff
@@ -1447,4 +1447,52 @@ b: Another tensor, of same type and shape as `a`.
 product: Pairwise cross product of the vectors in `a` and `b`.
 )doc");
 
+// --------------------------------------------------------------------------
+
+REGISTER_OP("ScanSum")
+    .Input("x: T")
+    .Attr("axis: int32")
+    .Output("out: T")
+    .Attr("T: numbertype")
+    .Doc(R"doc(
+Compute the cumulative sum of the input tensor `x` along the axis `axis`.
```

Comment:
I agree with @vrv `cumsum` is easier to remember and understand what it does. Your description also confirms that. To describe the operation you don't use 'scan' anywhere.